packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/Detail/AdvancedTxDetails/IODetails/TransactionDetailsModal.tsx

```
@@ -0,0 +1,118 @@
+                <CodeBlock $isDarkTheme={isDarkTheme}>
+                    <Translation id="TR_TXID" />:{'\n'}
+                    <Paragraph isMonospaced>{tx.txid}</Paragraph>
+                    {tx.amount && (
+                        <>
+                            {formatNetworkAmount(tx.amount, tx.symbol)} {formattedSymbol}
+                        </>
+                    )}
+                    {'\n'}
+                    {tx.details.vin.map((input, index) => (
+                        <>
+                            <Translation id="TR_INPUTS" /> {index + 1}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)